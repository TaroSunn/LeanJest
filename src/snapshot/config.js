export const config = {
  server: 'http://localhost',
  userName: 'test',
  password: '1234'
}