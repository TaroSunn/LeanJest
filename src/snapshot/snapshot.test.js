import { config } from "./config";
test("快照测试", () => {
  expect(config).toMatchInlineSnapshot(`
    Object {
      "password": "1234",
      "server": "http://localhost",
      "userName": "test",
    }
  `);
});
