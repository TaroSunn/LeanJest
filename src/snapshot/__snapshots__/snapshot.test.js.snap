// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`快照测试 1`] = `
Object {
  "password": "1234",
  "server": "http://localhost",
  "userName": "test",
}
`;
